aws/data_source_aws_resourcegroupstaggingapi_resources.go

```diff
@@ -0,0 +1,193 @@
+										Type:     schema.TypeBool,
+										Computed: true,
+									},
+									"keys_with_noncompliant_values": {
+										Type:     schema.TypeSet,
+										Computed: true,
+										Elem:     &schema.Schema{Type: schema.TypeString},
+									},
+									"non_compliant_keys": {
+										Type:     schema.TypeSet,
+										Computed: true,
+										Elem:     &schema.Schema{Type: schema.TypeString},
+									},
+								},
+							},
+						},
+						"tags": tagsSchemaComputed(),
+					},
+				},
+			},
+		},
+	}
+}
+
+func dataSourceAwsResourceGroupsTaggingAPIResourcesRead(d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*AWSClient).resourcegroupstaggingapiconn
+
+	input := &resourcegroupstaggingapi.GetResourcesInput{}
+
+	if v, ok := d.GetOk("include_compliance_details"); ok {
+		input.IncludeComplianceDetails = aws.Bool(v.(bool))
+	}
+
+	if v, ok := d.GetOk("exclude_compliant_resources"); ok {
+		input.ExcludeCompliantResources = aws.Bool(v.(bool))
+	}
+
+	if v, ok := d.GetOk("resource_arn_list"); ok && v.(*schema.Set).Len() > 0 {
+		input.ResourceARNList = expandStringSet(v.(*schema.Set))
+	}
+
+	if v, ok := d.GetOk("tag_filters"); ok {
+		input.TagFilters = expandAwsResourceGroupsTaggingAPITagFilters(v.([]interface{}))
+	}
+
+	if v, ok := d.GetOk("resource_type_filters"); ok && v.(*schema.Set).Len() > 0 {
+		input.ResourceTypeFilters = expandStringSet(v.(*schema.Set))
+	}
+
+	var taggings []*resourcegroupstaggingapi.ResourceTagMapping
+
+	err := conn.GetResourcesPages(input, func(page *resourcegroupstaggingapi.GetResourcesOutput, lastPage bool) bool {
+		if page == nil {
+			return !lastPage
+		}
+
+		taggings = append(taggings, page.ResourceTagMappingList...)
+		return !lastPage
+	})
+	if err != nil {
+		return err
+	}
+
+	d.SetId(meta.(*AWSClient).partition)
+
+	if err := d.Set("resource_tag_mapping_list", flattenAwsResourceGroupsTaggingAPIResourcesTagMappingList(taggings)); err != nil {
+		return fmt.Errorf("error setting resource tag mapping list: %w", err)
+	}
+
+	return nil
+}
+
+func expandAwsResourceGroupsTaggingAPITagFilters(filters []interface{}) []*resourcegroupstaggingapi.TagFilter {
+	result := make([]*resourcegroupstaggingapi.TagFilter, len(filters))
+
+	for i, filter := range filters {
+		m := filter.(map[string]interface{})
+
+		result[i] = &resourcegroupstaggingapi.TagFilter{
+			Key: aws.String(m["key"].(string)),
+		}
+
+		if v, ok := m["values"]; ok && v.(*schema.Set).Len() > 0 {
+			result[i].Values = expandStringSet(v.(*schema.Set))
+		}
+	}
+
+	return result
+}
+
+func flattenAwsResourceGroupsTaggingAPIResourcesTagMappingList(list []*resourcegroupstaggingapi.ResourceTagMapping) []map[string]interface{} {
+	result := make([]map[string]interface{}, 0, len(list))
+
+	for _, i := range list {
+		l := map[string]interface{}{
+			"resource_arn": aws.StringValue(i.ResourceARN),
+			"tags":         keyvaluetags.ResourcegroupstaggingapiKeyValueTags(i.Tags).IgnoreAws().Map(),
```

Comment:
Add `IgnoreConfig(ignoreTagsConfig)` (need to pass `ignoreTagsConfig` from `meta.(*AWSClient).IgnoreTagsConfig`).